apps/src/tutorialExplorer/responsive.jsx

```diff
@@ -1,6 +1,6 @@
 /* A very simple responsive layout system.
  */
-
+import $ from 'jquery';
 import * as utils from '../utils';
```

Comment:
How was this working without the jquery import? Was it working?

Reply:
This was working because we _almost_ always provide jQuery on window and can just assume that it's available everywhere.  In fact, [this bit of config](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/Gruntfile.js#L539-L543) means any time we import `'jquery'` we're actually just referring to the copy on window - in our regular build.

In unit tests we don't actually load jQuery onto window, and it has to be imported like a normal npm module - so when I wrote a unit test for this file, I was seeing failures about `$` not being defined.  Adding the import lets it work both in unit tests and in our regular build.